package.json

```diff
@@ -37,6 +37,7 @@
     "recompile": "gulp gitPreCompile && npm run bump && gulp gitPostCompile",
     "release": "gulp gitCreateRC",
     "test": "concurrently 'npm run test:prepare' 'sleep 5 && npm run test:run'",
+    "test:generators": "concurrently 'npm run test:prepare' 'sleep 25 && tests/scripts/run_generators.sh'",
```

Comment:
Why is the sleep now 25?

Reply:
At some point while i was testing the extra sleep was necessary for running generator tests. However, it looks like 5 seconds works for me locally, so I will change to 5.